src/storage/FileResourceStore.ts

```diff
@@ -350,21 +338,27 @@ export class FileResourceStore implements ResourceStore {
     }
     return new Promise(async(resolve, reject): Promise<any> => {
       try {
-        const stats = await fsPromises.lstat(path);
+        const stats = await fsPromises.lstat(`${this.rootFilepath}${path}`);
```

Comment:
I missed this last time, but you should be using the `path` library for anything related to file paths, so for this line you would have to use `path.join`. This reduces the chances of running into problems related to windows/unix file paths.